packages/blade/src/components/Badge/Badge.tsx

```diff
@@ -134,7 +134,7 @@ const _Badge = ({
         paddingLeft={horizontalPadding[size]}
         paddingTop={verticalPadding[size]}
         paddingBottom={verticalPadding[size]}
-        display="flex"
+        display={isReactNative() ? "flex" : "inline-flex"}
```

Comment:
```suggestion
        display={(isReactNative() ? "flex" : "inline-flex") as never}
```

This should fix the TS error. you can double check by running `yarn tsc:blade`

Reply:
Oh yes this is the weird error we randomly face sometimes because of faulty ts plugin. Let me take a pull and fix it.

Reply:
@archie252000 You can check and merge this PR into your branch - https://github.com/archie252000/blade/pull/1

I have fixed the build issue and updated the snapshot tests so checks should work fine with this.

Reply:
Yeah, it works. Build errors are gone. Thanks!